### ISSUE TITLE
a11y: question numbers are not read out when using Tab to navigate (Windows)

### ISSUE THREAD

**Issue by staceytan1998** [opengovsg/FormSG#5587]
**Issue**
When a user uses the Tab key to navigate from one form element to another, some questions do not have their question numbers read out. This issue does not appear when using the arrow keys to navigate instead.

This issue applies to all the questions on the sample forms except 3, 6, 7, and 11.

i.e. It affects the following form question types:

- Text box
- MCQ radio buttons
- MCQ checkbox
- Yes/No buttons
- Numerical input
- 1 to 5 stars rating
- File upload

**Suggestion from Etch**

The issue seems to occur due to incomplete ARIA labels used in the affected questions.

i.e. The affected questions do not have the question numbers included in the ARIA label for the question.

Including question numbers in the ARIA labels will fix this issue.

Removing the ARIA labels also does seem to fix this issue, since HTML <label> tags offer screen reader accessibility by default. This allows the screen reader to simply read the question text in the <label> tag, instead of allowing the ARIA label to take precedence.

**Screenshot**
https://drive.google.com/file/d/1zGETX_j1JYfxalFNICuYYu_zbu9QVVwC/view?usp=share_link

**Comment by foochifa** [comment]
Inconsistent when it comes to reading the question numbers with the actual questions.

Should follow the suggestions from Etch to use aria-label and append the question number and question together at the parent container.

**Comment by LinHuiqing** [comment]
Did some testing with my windows laptop and NVDA and found that the following field types are missing question numbers:

- Short answer
- Long answer
- Radio
- Checkbox
- Yes/no
- Rating
- Attachment
- Number
- Decimal
- NRIC
- UEN

Conversely, the following field types have question numbers read out:

- Dropdown
- Email
- Mobile number
- Home number
- Date
- Table

The screen record of this test is as shown below:

https://user-images.githubusercontent.com/37061143/219250751-7b563d6f-0f4f-4091-bcef-f476a60fe74e.mp4
